snmp.yml

```diff
@@ -1565,3 +1565,132 @@ fortinet_fortigate:
           type: Integer32
         - labelname: fgWcWtpSessionWtpId
           type: OctetString
+
+# ServerTech Sentry 3 MIB
+#
+# Used by ServerTech PDUs
+#
+# ftp://ftp.servertech.com/Pub/SNMP/sentry3/Sentry3OIDTree.txt
+# ftp://ftp.servertech.com/Pub/SNMP/sentry3/Sentry3.mib
+servertech_sentry3:
+  version: 2
+  auth:
+    community: public
+  walk:
+    - 1.3.6.1.2.1.1
+    - 1.3.6.1.4.1.1718.3.2.2.1
+    - 1.3.6.1.4.1.1718.3.2.3.1
+  metrics:
+    - name: sysUpTime
+      oid: 1.3.6.1.2.1.1.3
+    # Infeed metrics
+    - name: infeed_load_hundredths_amper
+      oid: 1.3.6.1.4.1.1718.3.2.2.1.7
+      indexes:
+        - labelname: towerIndex
+          type: Integer32
+        - labelname: infeedIndex
+          type: Integer32
+      lookups:
+        - labels: [towerIndex, infeedIndex]
+          labelname: infeedName
+          oid: 1.3.6.1.4.1.1718.3.2.2.1.3
+    - name: infeed_votage_tenths_volt
+      oid: 1.3.6.1.4.1.1718.3.2.2.1.11
+      indexes:
+        - labelname: towerIndex
+          type: Integer32
+        - labelname: infeedIndex
+          type: Integer32
+      lookups:
+        - labels: [towerIndex, infeedIndex]
+          labelname: infeedName
```

Comment:
Sure it can, we could automatically map all `DisplayString` items from the MIB.

```
    OutletEntry ::= SEQUENCE {
        outletIndex          INTEGER,
        outletID             DisplayString,
        outletName           DisplayString,
        outletCapabilities   BITS,
        outletStatus         INTEGER,
        outletLoadStatus     INTEGER,
        outletLoadValue      INTEGER,
        outletLoadLowThresh  INTEGER,
        outletLoadHighThresh INTEGER,
        outletControlState   INTEGER,
        outletControlAction  INTEGER,
        outletCapacity       INTEGER,
        outletVoltage        INTEGER,
        outletPower          INTEGER,
        outletApparentPower  INTEGER,
        outletPowerFactor    INTEGER,
        outletCrestFactor    INTEGER,
        outletEnergy         INTEGER,
        outletWakeupState    INTEGER,
        outletPostOnDelay    INTEGER
    }
```